default.n2r

```diff
@@ -4,34 +4,30 @@
 
 %global npm_name $NAME
 
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
 Name: nodejs-%{npm_name}
 Version: $VERSION
 Release: $RELEASE%{?dist}
 Summary: $SUMMARY
 License: $LICENSE
-Group: Development/Libraries
 URL: $PROJECTURL
 $SOURCES
-Requires: nodejs(engine)
-BuildRequires: nodejs-devel
 BuildRequires: nodejs-packaging
-BuildRequires: npm
-BuildArch: noarch
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
 ExclusiveArch: %{nodejs_arches} noarch
-Provides: npm(%{npm_name}) = %{version}
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
 $PROVIDES
 
 %description
-$DESCRIPTION
-
-%package doc
```

Comment:
Only large or multi-file documentation should go to separate doc subpackage, which is very rare in nodejs modules and I can't remember seeing it anywhere except npm.